content/_code-samples/delete-account/js/delete-account.js

```diff
@@ -0,0 +1,65 @@
+if (typeof module !== "undefined") {
```

Comment:
I'd recommend renaming this to `blackhole-account.js` instead of `delete-account.js` because it could be confusing for people who think this is a snippet for `AccountDelete`. (Same to the python file)